auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -362,10 +363,39 @@ def delete_affiliation_invitation(invitation_id):
         else:
             invitation.delete()
 
+    @staticmethod
+    def _filter_request_invites_role_based(affiliation_invitation_models: List[AffiliationInvitationModel],
+                                           org_id: int,
+                                           user: UserService = None) -> List[AffiliationInvitationModel]:
+        """Filter out affiliation invitations of type REQUEST if current user is not staff or org admin/coordinator."""
+        if UserService.is_context_user_staff():
+            return affiliation_invitation_models
+
+        if org_id is None:
+            return []
+
+        current_user: UserService = user or UserService.find_by_jwt_token()
```

Comment:
Look at the `@user_context` decorator?

https://github.com/bcgov/sbc-auth/blob/main/auth-api/src/auth_api/utils/user_context.py